ui/apps/profile/src/components/profile-stats-view.tsx

```diff
@@ -16,24 +18,33 @@ const ProfileStatsView: React.FC<ProfileStatsViewProps> = ({
   navigateTo,
   showLoginModal,
 }) => {
+  const [stats, setStats] = useState({
+    totalFollowing: 0,
+    totalFollowers: 0,
+    totalBeams: 0,
+    totalTopics: 0,
+  });
+
   const { t } = useTranslation('app-profile');
+  const { logger } = useRootComponentProps();
 
   const loginQuery = useGetLogin();
-  const isLoggedIn = React.useMemo(() => {
+  const isLoggedIn = useMemo(() => {
     return !!loginQuery.data?.id;
   }, [loginQuery.data]);
 
-  const statsQuery = useGetProfileStatsByDidQuery(
-    { id: profileId },
-    { select: response => response.node },
-  ) as { data: { akashaProfile: AkashaProfile } };
+  const sdk = getSDK();
 
-  const stats = {
-    beamsTotal: 10,
-    interestsTotal: 10,
-    followersTotal: statsQuery.data?.akashaProfile?.followersCount,
-    followingTotal: 10,
-  };
+  useEffect(() => {
+    const getStats = () => {
```

Comment:
I think you wanted here maybe to declare it async so you can use `await` instead of the promise api?

Reply:
Okay, I will refactor it

Reply:
you can also create a react-query hook for this call so you have caching